### DISCUSSION TITLE
Windows version crashes when creating the world

### DISCUSSION THREAD

**Discussion by Fewnity** [CrossCraft/CrossCraft-SurvivalTest#1] [General]
Hello, I tried the last commit to see AI, and the Windows version is not working :(
![image](https://user-images.githubusercontent.com/39272935/185406360-f9995fd0-9b71-4466-be4e-ad5b1439491c.png)

**Comment by IridescentRose** [comment]
You don't have the models installed most likely. Those were not included in the build yet -- I can add them in a moment.

**Reply by IridescentRose** [reply]
Fixed
